tools/js-sdk-release-tools/src/changelogToolCli.ts

```diff
@@ -1,16 +1,25 @@
 #!/usr/bin/env node
 
-import {generateChangelogAndBumpVersion} from "./common/changlog/automaticGenerateChangeLogAndBumpVersion.js";
-import {logger} from "./utils/logger.js";
+import { generateChangelogAndBumpVersion } from "./common/changlog/automaticGenerateChangeLogAndBumpVersion.js";
```

Comment:
also update this cli: https://github.com/Azure/azure-sdk-tools/blob/main/tools/js-sdk-release-tools/src/autoGenerateInPipeline.ts